src/meta/client/MetaClient.cpp

```diff
@@ -107,6 +110,35 @@ void MetaClient::stop() {
     isRunning_ = false;
 }
 
+void MetaClient::dnsQUeryThreadFunc() {
+    SCOPE_EXIT {
+        bgThread_->addDelayTask(
+            FLAGS_meta_domain_ttl_secs * 1000, &MetaClient::dnsQUeryThreadFunc, this);
```

Comment:
If metad ip changes, We will wait for 1000s to get right ipaddress, right?

Reply:
The parameter of addDelayTask is milliseconds, so here needs to be multiplied by 1000.